vllm/model_executor/model_loader/tensorizer.py

```diff
@@ -61,6 +61,14 @@
 
 logger = init_logger(__name__)
 
+def tensorizer_kwargs_arg(value):
+    loaded = json.loads(value)
+    if not isinstance(loaded, dict):
+        raise argparse.ArgumentTypeError(
+            f"Not deserializable to dict: {value}. serialization_kwargs and "
+            f"deserialization_kwargs must be "
+            f"deserializable from a JSON string to a dictionary. "
+        )
```

Comment:
This function is missing a return value...
If you were testing like this, it may have been switching the parsed value to `None` every time, so please test again with the corrected version.
```suggestion
def tensorizer_kwargs_arg(value):
    loaded = json.loads(value)
    if not isinstance(loaded, dict):
        raise argparse.ArgumentTypeError(
            f"Not deserializable to dict: {value}. serialization_kwargs and "
            f"deserialization_kwargs must be "
            f"deserializable from a JSON string to a dictionary. "
        )
    return loaded
```